config.yml

```diff
@@ -4,12 +4,16 @@ RunConfig:
 SearcherConfig:
   max_depth: 6 # maximum depth for negamax search
   search_mode: SINGLE_PROCESS # mode of search, choice of: SINGLE_PROCESS, LAZY_SMP
-  move_order_mode: COMPOSITE # mode of move ordering, choice of: MVV_LVA, KILLER_MOVE, COMPOSITE
   enable_null_move_pruning: True # flag to enable null move pruning, i.e. checking for a beta-cutoff by passing our turn
   enable_futility_pruning: False # flag to enable futility_pruning, i.e. prune quiet moves that do not raise alpha in negamax search.
   enable_delta_pruning: True # flag to enable delta pruning, i.e. prune capturing moves that do not raise alpha in quiescence search.
   enable_transposition_table: False # flag to enable position cache, so positions are not evaluated twice
   enable_aspiration_windows: True # flag to enable aspiration windows, i.e. doing a smaller window search around the score from previous iteration of iterative deepening.
+  # TODO: I'd like to make these capital to match class vs variable, but for later.
```

Comment:
As in you make to make the variable name all upper-cased? I thought upper case only for [constants](https://www.linkedin.com/pulse/write-python-code-looks-good-importance-pep-8-style-1-shah-hussain/) - not sure makes sense to use upper case in configs?

Reply:
For the yml file I'm following the Python convention that classes are CamelCase and variables are snake_case. The problem here is that when constructing SearcherConfig it looks for move_ordering_config instead of MoveOrderingConfig since we pass it as an argument to init class.

Maybe we can just use snake_case for all classes in yml file instead.